stacker/commands/stacker/info.py

```diff
@@ -0,0 +1,15 @@
+"""Gets information on the CloudFormation stacks based on the given config."""
+
+from .base import StackerCommand
+from ...actions import info
+
+
+class Info(StackerCommand):
+
+    name = 'info'
+    description = __doc__
+
+    def run(self, options, **kwargs):
+        super(Info, self).run(options, **kwargs)
+        action = info.Action(options.context, provider=options.provider)
```

Comment:
Also, might be cool to have a --output or --format argument.  Maybe not necessary here, but I could see someone wanting this in a machine readable format (like csv).